dss/events/handlers/index.py

```diff
@@ -372,14 +403,37 @@ def notify_subscriber(self, subscription: dict):
                                 f"Code: {response.status_code}")
 
 
-def create_elasticsearch_index(index_name: str, replica: Replica, logger: logging.Logger):
-    es_client = ElasticsearchClient.get(logger)
-    if not es_client.indices.exists(index_name):
-        with open(os.path.join(os.path.dirname(__file__), "mapping.json"), "r") as fh:
-            index_mapping = json.load(fh)
-        index_mapping["mappings"][ESDocType.doc.name] = index_mapping["mappings"].pop("doc")
-        index_mapping["mappings"][ESDocType.query.name] = index_mapping["mappings"].pop("query")
-        alias_name = Config.get_es_alias_name(ESIndexType.docs, replica)
-        create_elasticsearch_doc_index(es_client, index_name, alias_name, logger, index_mapping)
-    else:
-        logger.debug(f"Using existing Elasticsearch index: {index_name}")
+class BundleTombstoneDocument(IndexDocument):
+
+    def __init__(self, replica: Replica, bundle_uuid: str, bundle_version: typing.Optional[str],
+                 logger: logging.Logger) -> None:
+        super().__init__(replica, bundle_uuid, bundle_version, logger)
+
+    @classmethod
+    def from_bucket(cls, replica: Replica, bucket_name: str, key: str, logger):
```

Comment:
@mweiden said this should be `from_replica`

Reply:
✅